src/sphinxcontrib/bibtex/__init__.py

```diff
@@ -50,6 +55,19 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_directive("footbibliography", FootBibliographyDirective)
     app.add_role("footcite", FootCiteRole())
 
+    # Catch bug in newer docutils
+    if (0, 18, 0, 'final', 0, True) <= docutils.__version_info__ < (0, 20, 0, 'final', 0, True):
+        error = RuntimeError(
+            (
+                "sphinxcontrib-bibtex is known to be broken with the installed version of "
+                "docutils ({}). Once a fixed version is available, please update both "
+                "packages (sphinxcontrib-bibtex and docutils) to remove this error."
+            ).format(docutils.__version__)
+        )
+        SPHINX_LOGGER.critical(error.args[0])
+        raise error
```

Comment:
People who already run the extension with docutils 0.18 or 0.19 should be able to keep using the extension as before, in order not to cause a mass breakage. Is it possible just to issue a warning rather than raising an error?